mesonbuild/compilers/c.py

```diff
@@ -810,8 +822,13 @@ def has_multi_arguments(self, args, env):
 Using them to check linker arguments are never supported, and results
 are likely to be wrong regardless of the compiler you are using.
```

Comment:
This warning might usefully mention the name of the method you should be using now.

Reply:
Good point.